tycho-core/src/test/java/org/eclipse/tycho/core/testcomponents/StubP2DependencyResolver.java

```diff
@@ -1,21 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2008, 2011 Sonatype Inc. and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *    Sonatype Inc. - initial API and implementation
- *******************************************************************************/
-package org.eclipse.tycho.core.testcomponents;
-
-import org.codehaus.plexus.component.annotations.Component;
-import org.eclipse.tycho.core.DependencyResolver;
-import org.eclipse.tycho.core.osgitools.targetplatform.LocalDependencyResolver;
-
-// TODO romove me as part of TYCHO-527
-@Component(role = DependencyResolver.class, hint = "p2", instantiationStrategy = "per-lookup")
-public class StubP2DependencyResolver extends LocalDependencyResolver {
```

Comment:
@mickaelistria it seems the tests use this StubP2 resolver that actually was a LocalDependencyResolver, do you know why this was choosen instead of using the "real" p2 resolver?

Reply:
No idea sorry. If we can get rid of stubs without pitfalls, I'm all for it.

Reply:
I think the problem is that P2 requires tycho core and thus tycho core cant depend on P2 for test-purpose. That's a bit undesirable as we actually test something other than the P2 resolver here.

I'll check how much we can stub this or if I can just move tests around....